packages/nuxt-module/package.json

```diff
@@ -63,7 +63,7 @@
     "@nuxt/test-utils": "^3.19.2",
     "@types/node": "^22.16.3",
     "eslint": "^9.30.1",
-    "nuxt": "^3.17.6",
+    "nuxt": "^4.0.0",
```

Comment:
bumped this for test fixtures while I was at it. just released a few minutes ago!